index.js

```diff
@@ -269,7 +301,7 @@ class HomeBridgeTasmotaAirconHTTP {
       Celsius: normalizeOnOff(state.temperatureUnit != 'F'),
       Clean: normalizeOnOff(state.clean),
       Econo: normalizeOnOff(state.econo),
-      FanSpeed: String(Math.ceil(this.fanSteps * state.fanSpeed / 100) || 1), // Should never be zero, because it would mean "Off"
+      FanSpeed: this._valueOfFanForTasmota(state.fanSpeed, this.fanSteps, state.fanAutoSpeed),
```

Comment:
You don't have to pass inn the parameters since you can just pull them out of `this` inside the method.